src/main/java/org/prebid/server/bidder/pangle/model/WrappedImpExtBidder.java

```diff
@@ -18,4 +18,9 @@
 
     @JsonProperty("adtype")
     Integer adType;
+
+    @JsonProperty("is_prebid")
```

Comment:
as default we use snake_case so this one will be converted to is_prebid, without @JsonProperty